src/server.ts

```diff
@@ -249,6 +263,18 @@ const get_metadata = (ecosystem, name, version) => {
         }
     });
 };
+/* Total Counts of #Known Security Vulnerability and #Security Advisory */
+class TotalCount 
+{
+    vulnerabilityCount: number;
+    advisoryCount: number;

```

Comment:
You can directly assign the initial value without constructor.

```
vulnerabilityCount: number = 0;
advisoryCount: number = 0;
```

Reply:
@arajkumar PTAL : https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/commit/0391ba4a038ba374107d78e1952d7d2b92367ab8